src/VideoUtils.ts

```diff
@@ -151,14 +151,9 @@ export function createUniquePath(videos: Array<Video>, outDirs: Array<string>, t
         while (!skip && fs.existsSync(path.join(outDirs[index], finalTitle + '.' + format))) {
             finalTitle = `${title}.${++i}`;
         }
-
-
-        if ("win32" == (require('os').platform() || "win32")) { // 
-            video.outPath = path.join(outDirs[index], `${sanitizeWindowsName(finalTitle, { replacement: "_" })}.${format}`);
-        } else {
-            video.outPath = path.join(outDirs[index], finalTitle + '.' + format);
-        }
-
+       
+        video.outPath = path.join(outDirs[index], `${sanitizeWindowsName(finalTitle, { replacement: "_" })}.${format}`);
```

Comment:
consistent naming on all platforms

Reply:
we were sanitizing the filename because often they have name with "" or similar and we just want to remove them, not substitute with an underscore

Reply:
ok so a first pass with removing for title, and then a second pass for final name with replacing should be correct ?

Reply:
please see the new review on why I don't think we should sanitize but error out